### ISSUE TITLE
NEP global variable ignored

### ISSUE THREAD

**Issue by Phate4569** [soolar/sl_ascend#273]
On ascension days I do the NEP quest to get Drinking TO Drink. In my last iteration I received the "Party Hard" shirt, so I needed the script to ignore NEP (as I do with Pirate and Fantasy on ascension days). I turned "neverendingPartyAlways" to false in the relay, yet the script still performed its daily adventures there.

See screenshot attached.
![DoesNEPWithoutPermission](https://user-images.githubusercontent.com/51530157/61170524-86cf6080-a538-11e9-89ad-b57fb6bcb5c7.png)

**Comment by Phate4569** [comment]
New run, same issue. It is still doing it making me believe it is not another one of my flukes.

**Comment by Phate4569** [comment]
Ok this is driving me INSANE.
Before running the script:
rugrat@KipLaptop:~/.kolmafia/settings$ grep neverendingPartyToday *
boondoggle_the_magnificent_prefs.txt:_neverendingPartyToday=false

After running the script:
rugrat@KipLaptop:~/.kolmafia/settings$ grep neverendingPartyToday *
boondoggle_the_magnificent_prefs.txt:_neverendingPartyToday=true

I can't find anything that sets this. Is it a mafia thing?

**Comment by Phate4569** [comment]
Nope, Mafia's default setting is "false"....

**Comment by Phate4569** [comment]
Ezandora enlightened me:

Ezandora: set it to false, then visit the wrong side of the tracks. then check it again
Ezandora: it means "you have the party unlocked today"

Mafia automatically checks.

**Comment by Phate4569** [comment]
So...Here the rub.

neverendingPartyAlways = This is manually set.
neverendingPartyToday = Ignored unless "neverendingPartyAlways=false" then on the first visit to wrong side of the tracks it sets this variable to "true"

Together these conspire to make the script ALWAYS adventure at NEP.

**Comment by jaspercb** [comment]
I suggest you edit the source to always skip NEP.

**Comment by Phate4569** [comment]
Yeah, I'm going to edit my local copy to just use neverendingPartyAlways only.

It is just annoying that there is no way for a standard user to "Turn Off" NEP so that they can farm the quests in Aftercore.
